openhands_cli/acp_impl/confirmation.py

```diff
@@ -0,0 +1,169 @@
+"""Confirmation mode implementation for ACP."""
+
+import logging
+from collections.abc import Callable
+from typing import Literal
+
+from acp import Client
+from acp.schema import (
+    AllowedOutcome,
+    PermissionOption,
+    ToolCallUpdate,
+)
+
+from openhands.sdk.event import ActionEvent
+from openhands.sdk.security.confirmation_policy import ConfirmRisky, NeverConfirm
+from openhands.sdk.security.risk import SecurityRisk
+from openhands_cli.user_actions.types import ConfirmationResult, UserConfirmation
+
+
+logger = logging.getLogger(__name__)
+
+
+# Type alias for confirmation modes
+ConfirmationMode = Literal["always-ask", "always-approve", "llm-approve"]
```

Comment:
Would be great to double check we're using the same literals for everywhere else in the CLI 

Doesn't have to be this PR tho!